### ISSUE TITLE
Error when renaming Organisation

### ISSUE THREAD

**Issue by D-04** [go-gitea/gitea#13604]
- Gitea version (or commit ref): 1.14.0+dev-231-g75ebf7c5b
- Git version:
- Operating system: latest docker image (ecb484e3968c) on linux amd64
  
  
  
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL) [repo_before_rename](https://try.gitea.io/orgarenametest/repo) [repo_after_rename](https://try.gitea.io/orgarenametestt/repo)
  - [ ] No
- Log gist:

When editing the orga name:
```
2020/11/17 15:16:07 Completed GET /user/events 0  in 9.762082211s
2020/11/17 15:16:07 Started POST /org/testt/settings for 172.17.0.1:54722
2020/11/17 15:16:07 ...uters/org/setting.go:94:SettingsPost() [E] UpdateUser: e-mail invalid [email: ]
2020/11/17 15:16:07 Completed POST /org/testt/settings 0  in 16.868602ms
2020/11/17 15:16:07 Started GET /user/avatar/test/-1 for 172.17.0.1:54722
2020/11/17 15:16:07 Completed GET /user/avatar/test/-1 0  in 2.331973ms
2020/11/17 15:16:08 Started GET /user/events for 172.17.0.1:54722
2020/11/17 15:16:08 Started GET /serviceworker.js for 172.17.0.1:54728
2020/11/17 15:16:08 ...les/public/public.go:158:handle() [I] [Static] Serving /serviceworker.js
2020/11/17 15:16:08 Completed GET /serviceworker.js 0  in 473.256µs
```
When accessing the old repository after the orga rename:
```
2020/11/17 15:17:48 Completed GET /user/events 0  in 4.665781158s
2020/11/17 15:17:48 Started GET /oldorga/repo for 172.17.0.1:54744
2020/11/17 15:17:48 ...ules/context/repo.go:526:func1() [E] RepoAssignment Invalid repo /data/git/repositories/oldorga/repo.git: no such file or directory
2020/11/17 15:17:48 Completed GET /oldorga/repo 0  in 9.783492ms
2020/11/17 15:17:48 Started GET /user/avatar/test/-1 for 172.17.0.1:54744
2020/11/17 15:17:48 Completed GET /user/avatar/test/-1 0  in 1.216531ms
2020/11/17 15:17:48 Started GET /user/events for 172.17.0.1:54744
2020/11/17 15:17:49 Started GET /serviceworker.js for 172.17.0.1:54760
2020/11/17 15:17:49 ...les/public/public.go:158:handle() [I] [Static] Serving /serviceworker.js
2020/11/17 15:17:49 Completed GET /serviceworker.js 0  in 283.299µs
```
## Description
When renaming an organisation with repositories the server responds with a 500 error (logs above) and the repositories are not available under the new URL (404 error). If you try accessing the repo under the old URL you will get a 500 error (logs above).

This has happened on both PostgreSQL and SQLite

## Screenshots
Creating a new Organisation
![1](https://user-images.githubusercontent.com/51401275/99411675-2a324180-28f4-11eb-823d-890dcc1f1388.png)

Creating a new Repository inside this Organisation
![2](https://user-images.githubusercontent.com/51401275/99411689-2bfc0500-28f4-11eb-9a9d-2787dfedef94.png)

Renaming the Organisation
![3](https://user-images.githubusercontent.com/51401275/99411694-2d2d3200-28f4-11eb-8500-28f565147d98.png)

500 Error after pressing Update Settings
![4](https://user-images.githubusercontent.com/51401275/99411702-30282280-28f4-11eb-84b4-98efc5f8713a.png)

New repository URL is being displayed in the Dashboard
![4 5](https://user-images.githubusercontent.com/51401275/99411698-2e5e5f00-28f4-11eb-839a-4568ccf101e6.png)

404 Error when accessing the new repository URL displayed in the Dashboard
![5](https://user-images.githubusercontent.com/51401275/99411709-31594f80-28f4-11eb-8453-281c7e9035d7.png)

500 Error when accessing the old URL
![6](https://user-images.githubusercontent.com/51401275/99411713-328a7c80-28f4-11eb-9885-c9500a986ca1.png)

**Comment by ventotto** [comment]
Can I try to resolve it?

**Comment by 6543** [comment]
@ventotto sure

**Comment by noerw** [comment]
@ventotto anythin new?

**Comment by D-04** [comment]
This seems to have been fixed by the 1.14 release.
